src/Sentry.Unity.Editor/Android/AndroidManifestConfiguration.cs

```diff
@@ -116,6 +117,59 @@ public void OnPostGenerateGradleAndroidProject(string basePath)
             // TODO: All SentryOptions and create specific Android options
 
             _ = androidManifest.Save();
+
+            AppendSymbolUploadToGradleProject(basePath, options);
+        }
+
+        internal void AppendSymbolUploadToGradleProject(string basePath, SentryUnityOptions options)
+        {
+            var cliOptions = SentryCliOptions.LoadCliOptions();
+            if (!cliOptions.UploadSymbols || EditorUserBuildSettings.development)
+            {
+                return;
+            }
```

Comment:
We plan an opt-in for the first condition, right? So we can upload dev build symbols if the user selects that?

```suggestion
            if (EditorUserBuildSettings.development)
            {
                options.DiagnosticLogger?.LogDebug($"Development build. Not adding Android symbol upload to gradle.");
                return;
            }
            var cliOptions = SentryCliOptions.LoadCliOptions();
            if (!cliOptions.UploadSymbols)
            {
                options.DiagnosticLogger?.LogDebug($"UploadSymbols is disabled. Not adding Android symbol upload to gradle");
                return;
            }
```

Reply:
I thought about splitting that into 2 flags:
- Upload Symbols (at all)
- Upload Debug Symbols